disputer/disputer.js

```diff
@@ -1,5 +1,10 @@
 const { createObjectFromDefaultProps } = require("../common/ObjectUtils");
 const { revertWrapper } = require("../common/ContractUtils");
+const { PostWithdrawLiquidationRewardsStatusTranslations } = require("../common/Enums");
+const { interfaceName } = require("../core/utils/Constants");
+
+const Finder = artifacts.require("Finder");
```

Comment:
I like the DVMClient idea

Reply:
I thought we started not using artifacts because artifacts weren't being injected into files that weren't directly passed to `truffle exec`? Does this work now?

Reply:
Using artifacts with `truffle exec` does seem to work

Reply:
To be more explicit, the problem we used to have (around the time we started writing the EMP client), was when a runnable script imported another file and that file used `artifacts` or `web3`. We seemed to find that these were only injected into the entry script passed directly to `truffle exec` and that any files that were imported weren't seeing that injection.

It's weird that this is now working, but assuming it is, IMO, we should treat contracts like any other imports and just put `artifacts.require()` at the top of any file that uses them rather than trying to pass them around. The only caveat is if we want to change the contract that's passed in in tests. In that case, dependency injection could make sense.

Thoughts?

Reply:
I can still reproduce this issue. If I create a script that imports another file that tried to use `artifacts` or `web3`, the second file cannot find them. I'm not sure if this workaround is reasonable, but I've found that the following seems to work if you put it at the top of the main script (before it calls other scripts):
```
global.web3 = web3;
global.artifacts = artifacts;
```

This effectively shares those objects with all the other files that are running. Not sure if this has any bad consequences elsewhere? Hopefully no truffle components assume that these variables are _not_ defined globally.

Reply:
@mrice32 @chrismaree  I ended up injecting `votingContract`. I opted not to go with the DVMClient implementation because for now there is only one property I am interested in--the DVM Contract object, and specifically the `getPrice()` method.

If we want to add other properties, we can wrap into a DVMClient in a follow up PR.

Reply:
Your memory and testing of this behavior align with my understanding @mrice32.
@nicholaspai I'm happy with the implementation you have now because you inject the contract instance from the top level in the `index.js`. This way only this file needs to require the artifacts from truffle.